erpnext/patches/v13_0/disable_ksa_print_format_for_others.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2020, Wahni Green Technologies and Contributors
+# License: GNU General Public License v3. See license.txt
+
+import frappe
+
+
+def execute():
+	company = frappe.get_all('Company', filters = {'country': 'Saudi Arabia'})
+	if company:
+		return
+
+	if frappe.db.exists('DocType', 'Print Format'):
+		frappe.reload_doc("regional", "print_format", "ksa_vat_invoice", force=True)
+		frappe.reload_doc("regional", "print_format", "ksa_pos_invoice", force=True)
+		frappe.db.sql("""UPDATE`tabPrint Format` SET disabled = 1
```

Comment:
@rtdany10 Even version-13 supports query builder, so this query can be rewritten using query builder

Reply:
Couldn't get `frappe.qb.update()` to work. Used `frappe.db.set_value()` instead.